dev/PushNotifications/PushManager.cpp

```diff
@@ -4,11 +4,111 @@
 #include "pch.h"
 #include "PushManager.h"
 #include "PushManager.g.cpp"
+#include <winrt\windows.networking.pushnotifications.h>
+#include <winerror.h>
+#include <algorithm>
+#include <list>
+#include <iostream>
+#include <ChannelResult.h>
+#include <winerror.h>
+
+using namespace winrt::Windows::Networking::PushNotifications;
+using namespace winrt;
+
+// MAX BACKOFF TIME IS 16MINUTES 
+#define MAX_BACKOFF_SECONDS 960
+
+// Error code - platform is not connected
+#define WNP_E_NOT_CONNECTED 0x880403E8L
+
+// Error code - platform reconnecting with wns
+#define WNP_E_RECONNECTING  0x880403E9L
+
+// Error code - Current bind operation failed because another bind operation in progress
+#define WNP_E_BIND_USER_BUSY 0x880403FEL
```

Comment:
Would consider perhaps moving the maximum backoff value into a `const int` (or appropriate type) and localize it to the function its used in below. (Doesn't seem to be used elsewhere.)

Also curious if those errors are already defined in `winrt\windows.networking.pushnotifications.h` and if not, should they be moved there?

Finally, I don't think the comments are adding much value here. I would delete but I defer to coding style guide.

Reply:
Most of the above error codes weren't part of the winerror.h file I included above. So I had to introduce them here. Sure I will get rid fo the comments. I wanted to be clear. No these errors are not part of the winrt\windows.networking.pushnotifications.h. Thanks.

Reply:
@riverar Do you believe that the comments need to be more useful in some sort of way or to get rid of them altogether?

Reply:
They just seem to reuse the same words in the constant name itself, so I'd delete them. But I understand style guidelines and culture may dictate otherwise. Pick a path and run fast. 👍

Reply:
Gotcha. @sharath2727 Since these error codes are internal error codes and need a deep understanding on how the platform itself works, I would remove the comments. Otherwise, we need to explain what these actually are and I suspect that those are not one liners :)